src/components/Menu/Menu.examples.story.tsx

```diff
@@ -131,3 +131,29 @@ const PopList = (
     </Pop.List>
   </>
 );
+
+export function WithDisabledItems() {
```

Comment:
Nit: Can you pass an `{args}` here and spread it in your `<Menu>`? Just want to make sure a code snippet gets generated in the Docs tab (related PR: https://github.com/vimeo/iris/pull/156)